packages/codemods/src/v5/keep-previous-data/__testfixtures__/default.input.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react'
+import axios from 'axios'
+import { useQueries, useQuery } from '@tanstack/react-query'
+
+type Post = {
+  id: number
+  title: string
+  body: string
+}
+
+export const Example1 = () => {
+  const queryFn = async (): Promise<Array<Post>> => {
+    const { data } = await axios.get(
+      'https://jsonplaceholder.typicode.com/posts',
+    )
+    return data
+  }
+  const { data } = useQuery({
+    queryKey: ['posts'],
+    queryFn: queryFn,
+    keepPreviousData: true,
```

Comment:
I would only touch this if we have explicitly set `keepPreviousDat: true`. Because things like `keepPreviousData: someCondition` are harder to migrate. So we might need an example that proves that these are not touched and get a proper warning message.

Reply:
So it should be executed only when the `keppPreviousData` property has value `true`, right?

Reply:
9fd29542826b96b968dc0d75d2a90c475d01bf33